modules/nf-core/blast/tblastn/main.nf

```diff
@@ -28,7 +28,12 @@ process BLAST_TBLASTN {
         gzip -c -d ${fasta} > ${fasta_name}
     fi
 
-    DB=`find -L ./ -name "*.nsq" | sed 's/\\.nsq\$//'`
+    DB=`find -L ./ -name "*.nal" | sed 's/\\.nal\$//'`
```

Comment:
This actually more consistent among the two modules now 😄

Reply:
Yep ! 🚀